pkg/decision/persisting_experiment_service_test.go

```diff
@@ -53,12 +56,14 @@ func (s *PersistingExperimentServiceTestSuite) SetupTest() {
 	s.testComputedDecision = ExperimentDecision{
 		Variation: &computedVariation,
 	}
-	s.mockExperimentService.On("GetDecision", s.testDecisionContext, testUserContext).Return(s.testComputedDecision, nil)
+	s.options = decide.OptimizelyDecideOptions{}
+	s.reasons = decide.NewDecisionReasons(s.options)
+	s.mockExperimentService.On("GetDecision", s.testDecisionContext, testUserContext, s.options, s.reasons).Return(s.testComputedDecision, nil)
 }
 
 func (s *PersistingExperimentServiceTestSuite) TestNilUserProfileService() {
```

Comment:
Not sure if it's planned to add later, but we need test cases to validate "IgnoreUPS" option here.

Reply:
yes these tests are included in the succeeding PR. #299